src/main/java/pl/psnc/dei/ui/components/SearchResultsComponent.java

```diff
@@ -416,21 +416,16 @@ public void selectAll() {
     }
 
     public void inverseSelection() {
-        List<SearchResult> notSelected = searchResults.getResults().stream()
-                .filter(s -> !currentUserRecordSelection.getSelectedRecordIds().contains(s.getId()))
-                .collect(Collectors.toList());
-
-        List<SearchResult> selected = searchResults.getResults().stream()
-                .filter(s -> currentUserRecordSelection.getSelectedRecordIds().contains(s.getId()))
-                .collect(Collectors.toList());
         resultsList.removeAll();
-        notSelected.forEach(ns -> {
-            resultsList.add(createResultComponent(ns, true));
-            currentUserRecordSelection.addSelectedRecordId(ns.getId());
-        });
-        selected.forEach(s -> {
-            resultsList.add(createResultComponent(s, false));
-            currentUserRecordSelection.removeSelectedRecordId(s.getId());
-        });
+        for(SearchResult s :searchResults.getResults()){
+            if(!currentUserRecordSelection.getSelectedRecordIds().contains(s.getId())){
```

Comment:
You can still use streams to do it if you want: searchResults.getResults().forEach(s -> ...); but whatever option you choose please fix the formatting ("if () {", "} else {", "for () {").